ports/geckolib/properties.mako.rs

```diff
@@ -844,6 +844,40 @@ fn static_assert() {
     fn copy__moz_binding_from(&mut self, other: &Self) {
         unsafe { Gecko_CopyMozBindingFrom(&mut self.gecko, &other.gecko); }
     }
+
+    fn set_page_break_before(&mut self, v: longhands::page_break_before::computed_value::T) {
+        use style::computed_values::page_break_before::T;
+        let result = match v {
```

Comment:
hm... how does gecko discern between different values of the property if this is just a boolean for them? There should be something else to check it seems, could you elaborate?